Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -107,6 +108,10 @@ void initEntityPropertyEditorResources()
 namespace AzToolsFramework
 {
     static const char* kComponentEditorIndexMimeType = "editor/componentEditorIndices";
+    static const char* kComponentEditorRowWidgetType = "editor/componentEditorRowWidget";
+
+    static const QString kPropertyEditorMenuActionMoveUp("editor/propertyEditorMoveUp");
+    static const QString kPropertyEditorMenuActionMoveDown("editor/propertyEditorMoveDown");
```

Comment:
These should all be constexpr const char*. I am not sure if QString does anything special with (de)allocation or (de)construction

Reply:
Done